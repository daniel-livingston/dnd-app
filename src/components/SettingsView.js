import React from "react";
import View from "./View";
import SettingsForm from "./SettingsForm";

const SettingsView = () => (
	<View title='Settings' description='Customize your settings.'>
		<SettingsForm />
	</View>
);

export default SettingsView;
