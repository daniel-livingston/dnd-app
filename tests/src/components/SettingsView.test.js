import React from "react";
import { shallow } from "enzyme";
import SettingsView from "../../../src/components/SettingsView";

describe("<SettingsView>", () => {
	test("renders correctly", () => {
		const wrapper = shallow(<SettingsView />);
		expect(wrapper).toMatchSnapshot();
	});
});
