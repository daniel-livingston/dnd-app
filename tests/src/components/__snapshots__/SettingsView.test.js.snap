// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SettingsView> renders correctly 1`] = `
<Connect(View)
  description="Customize your settings."
  title="Settings"
>
  <Connect(SettingsForm) />
</Connect(View)>
`;
